base/reflection.jl

```diff
@@ -1150,13 +1150,18 @@ function func_for_method_checked(m::Method, @nospecialize(types), sparams::Simpl
 end
 
 """
-    code_typed(f, types; optimize=true, debuginfo=:default)
+    code_typed(f, types; kw...)
 
 Returns an array of type-inferred lowered form (IR) for the methods matching the given
-generic function and type signature. The keyword argument `optimize` controls whether
-additional optimizations, such as inlining, are also applied.
-The keyword `debuginfo` controls the amount of code metadata present in the output,
+generic function and type signature.
+
+# Keyword Arguments
+
+- `optimize=true`: controls whether additional optimizations, such as inlining, are also applied.
+- `debuginfo=:default`: controls the amount of code metadata present in the output,
 possible options are `:source` or `:none`.
+- `world=Base.get_world_counter()`: controls the world age to use when looking up methods.
+- `interp=Core.Compiler.NativeInterpreter(world)`: controls the interpreter to use.
```

Comment:
Are these two (`Base.get_world_counter()` and `Core.Compiler.NativeInterpreter`)  referenced from anywhere else? Otherwise, it seems a bit odd to have it in a docstring like this.

Reply:
I don't think they are documented in other places yet, should I just remove them from here then?

Reply:
Or should I also add a docstring for them?

Reply:
Please add a docstring for them as well